### ISSUE TITLE
Problem if page is longer than 8192px

### ISSUE THREAD

**Issue by richtrefz** [sindresorhus/capture-website#59]
If the rendered website page is longer than 8192 px, the .png file is nor right.  At 8192 px down, the top of the web page repeats

attachment is a very shrunken capture of https://api.jquery.com/ 
but you can see the repeated header

![jq](https://user-images.githubusercontent.com/1500959/96742731-3aa4da00-1391-11eb-9e60-d2c71b7c4fce.png)

**Comment by brandon93s** [comment]
Apologies for the thread-bump but confirming this behavior on all sites.  As well, across both Linux and Windows. 
 I hope to investigate the root cause soon.

![image](https://user-images.githubusercontent.com/4730164/97826590-fdddba80-1c8f-11eb-9d58-79f1a70bc8b8.png)

**Comment by krnik** [comment]
Using puppeteer only gives the same results:
```
	const browser = await puppeteer.launch();
	const page = await browser.newPage();
	await page.goto('https://api.jquery.com/');
	const screenshot = await page.screenshot({
		fullPage: true
	});

	require('fs').writeFileSync('test.png', screenshot);
	await page.close();
```

**Comment by brandon93s** [comment]
Appears to be a Chromium issue:  https://bugs.chromium.org/p/chromium/issues/detail?id=1043959&q=screenshot%20fullPage&can=2.  The only reliable workaround I've seen is to take a bunch of partial screenshots as you scroll and then stitch them back together.

**Comment by tthsympa** [comment]
Hi @krnik, did you actually succeed using firefox along capture-website? If yes, i'm curious about how you do that

**Comment by krnik** [comment]
Yes, it did succeed. But I used only one option `fullPage` as in the example above. Maybe that's the problem here.
